spec/models/king_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  describe '#create' do
+    it 'creates a king in the database' do
+      king = FactoryGirl.create(:king)
+      expect(king.type == 'King').to eq(true)
+    end
+  end
+
+  describe '#valid_move?' do
+    it 'returns true if the move is diagonal up-right one square' do
+      king = FactoryGirl.create(:king, x_position: 4, y_position: 4)

```

Comment:
Went ahead and saved my revision of `king_spec.rb` with the **king =** line shortened in each test as shown in my previous post. But I haven't resolved [the conflicts GitHub is citing](https://github.com/firehose-codemasters/codemasters_chess/pull/25/conflicts).

Which should I do first: resolve the conflicts or commit and push the change I just made to `king_spec.rb`?